---
geometry_msgs/PointStamped result
